### PR TITLE
Feat:Landing page and context

### PR DESCRIPTION
**Pull request by shrushti2000**
Added basic landing page and context
image
image

### CODE REVIEW

File: src/pages/Homepage/Homepage.js [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+        </div>
+      </div>
+      <div className='categories-section flex-vt'>
+        {state.categories.map(category => {
+          return (
+            <div className='category flex-hz jc-sa'>
+              <img className='category-img' src={category.img} alt="category-img" />
+              <div>
+                <p className='category-text text-md w-30'>{category.description}</p>
+                <button class="btn btn-outline-info category-btn">Explore more</button>
```
Comment (tushargoyal812) [inline src/pages/Homepage/Homepage.js:25]: you can use destructuring here

**Review by tushargoyal812 — COMMENTED** [review]
code is very clean and to the point.
variable naming is very good.
file structure is also good.

**Comment by SRUJANA-PENUGONDA13** [comment]
Everything looks good to me. Good file structuring

**Review by poojakelkar — APPROVED** [review]
Looks good to merge

**Pull request merged**